docs/source/use_CA.md

```diff
@@ -0,0 +1,205 @@
+# Using a CA
+
+If you've read our topics on [identity](https://hyperledger-fabric.readthedocs.io/en/release-2.1/identity/identity.html) and [Membership Service Provider (MSP)](https://hyperledger-fabric.readthedocs.io/en/release-2.1/membership/membership.html) you're aware that in Hyperledger Fabric, Certificate Authorities are used to generate the identities assigned to admins and nodes. While any Certificate Authority that can generate x.509 certificates can be used to create the public/private key pair that constitutes an identity, the Fabric CA can additionally generate the local and organizational MSP folder structures that are used by Hyperledger Fabric.
+
+In this topic, we'll show a "happy path" for using the Fabric CA to generate identities and MSPs. Note that you do not have to use the Fabric CA to register and enroll identities. However, if you use a different CA, you will need to create the relevant identities and MSPs that Fabric uses to build the configuration files of organizations and nodes. We will show examples of those MSPs below.
+
+## Overview of registration and enrollment
+
+While it is possible for the admin of a CA to create an identity and give the public/private key pair to a user out of band, this process would give the CA admin access to the private key of every user. Such an arrangement violates basic security procedures regarding the security of private keys, which should not be exposed for any reason.
+
+As a result, CA admins **register** users, a process in which the CA admin gives a username and password to an identity and assigns it a role and any required attributes. The CA admin then gives this username and password to the ultimate user of the identity. The user can then execute a Fabric CA client command using this username and password, returning the public/private certificate pair containing the role and attributes assigned by the CA admin.
+
+This process preserves both the integrity of the CA (because only CA admins can register users and assign roles and affiliations) and private keys (since only the user of an identity will have access to them).
+
+**While admin identities only need to be registered and enrolled with the "organization CA", the CA that generates identity certificates for admins and nodes, nodes must also be registered and enrolled with a TLS CA. This will create a public/private TLS key pair that nodes use to sign and encrypt their communications.** If the TLS CA has been created only using a "TLS" profile, the commands to register and enroll an identity are identical to those to register and enroll with the TLS CA. For more information about creating a CA that can only function as a TLS CA, check out [deployment guide link].
```

Comment:
Right --- this was a note for the scenario in which they have to specify a profile, in which case the command for generate TLS certs would be different because they would have to give the profile flag.